src/ramses_tx/parsers.py

```diff
@@ -2115,7 +2115,7 @@ def parser_31d9(payload: str, msg: Message) -> dict[str, Any]:
 
     bitmap = int(payload[2:4], 16)
 
-    # NOTE: 31D9[4:6] is fan_rate (itho?) *or* fan_mode (orcon?)
+    # NOTE: 31D9[4:6] is fan_rate (minibox!, itho?) *or* fan_mode (orcon?)
     result = {
         **parse_exhaust_fan_speed(payload[4:6]),  # itho
```

Comment:
> Is it OK if we use _22F1_MODE_ITHO from parsers to map these values to something more meaningful?

I am not exactly sure what you're asking - please provide more information.

Reply:
At the moment, the `parse_exhaust_fan_speed` helper returns simply the value it was given. Using a lookup on the mode-list we could return a state name instead.

Reply:
We have that scheme/lookup list here in ramses_rf, but for i18n/localization it might be best to wait for ramses_cc to assign the strings used in the UI